packages/entities/entities-plugins/src/composables/usePluginHelpers.ts

```diff
@@ -154,8 +154,8 @@ export default function useHelpers() {
   /**
    * Get the height of the tallest card
    */
-  const getTallestPluginCardHeight = (elements: Array<InstanceType<typeof PluginSelectCard>>): number => {
-    let tallestCardHeight = 0
+  const getTallestPluginCardHeight = (elements: Array<InstanceType<typeof PluginSelectCard>>, minHeight: number = 0): number => {
```

Comment:
Since we're passing `minHeight = 310` for `getTallestPluginCardHeight` in every reference, could we make it default in this function as `minHeight: number = 310`

Reply:
@TT1228 I modified the implementation a bit, I think we shouldn't modify the function `getTallestPluginCardHeight`, I use `Math.max(xxx, minHeight)` inside in component to ensure the minimum height